src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1045,4 +1051,47 @@ public static Font getTooltipFont(@Nullable Font forcedFont, @Nonnull ItemStack
                 .toList();
     }
 
+    public static void createWorldConfirmationScreen(
+            LevelStorageSource save, String worldName, boolean creatingWorld,
+            Function4<LevelStorageSource.LevelStorageAccess, RegistryAccess.RegistryHolder, ResourceManager, DataPackConfig, WorldData> f4,
+            Function<Function4<LevelStorageSource.LevelStorageAccess, RegistryAccess.RegistryHolder, ResourceManager, DataPackConfig, WorldData>, Runnable> runAfter)
+    {
+        Component title = new TranslatableComponent("selectWorld.backupQuestion.experimental");
+        Component msg = new TranslatableComponent("selectWorld.backupWarning.experimental");
+
+        Screen screen = new ConfirmScreen(confirmed ->
+        {
+            if (confirmed)
+            {
+                //The WorldData is re-created when re-running the runnable,
+                // so make sure to be setting the field to true on the right instance.
+                runAfter.apply((a,b,c,d) ->
+                {
+                    WorldData worldData = f4.apply(a,b,c,d);
```

Comment:
Could these lambada parameters please be given more descriptive names? Also, formatting: spaces after commas in parameter lists.

Reply:
I'd prefer not to, I am just using expanding the function a bit, when just doing `apply(f4)`, the parameters wouldn't be seen. IMO, giving these descriptive names will lead to people wanting/trying to understand them, while there is nothing to understand, they are just being used to be fed into the Function4.

but i understand if you still want me to describe them